From 538323aab5cb977572a86d14b4338a323a521da2 Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Thu, 25 Aug 2016 15:35:18 -0700
Subject: [PATCH] TOOLS-1535 gerrit could be better with names and faces
 Reviewed by: Alex Wilson <alex.wilson@joyent.com> Reviewed by: Dave Pacheco
 <dap@joyent.com> Approved by: Dave Pacheco <dap@joyent.com>

---
 images/appserver/etc/gerrit.config.base | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/images/appserver/etc/gerrit.config.base b/images/appserver/etc/gerrit.config.base
index d1e6159..70e32b8 100644
--- a/images/appserver/etc/gerrit.config.base
+++ b/images/appserver/etc/gerrit.config.base
@@ -37,6 +37,16 @@
 [cache]
 	directory = cache
 
+#
+# The "web_sessions" cache tracks open user sessions.  These effectively map a
+# cookie in your browser back to an identity.  The maximum age of entries in
+# this cache determines how often a user must sign in again.  A value of 0
+# allows entries to remain in the cache until it fills, at which point the
+# oldest entries will be purged.
+#
+[cache "web_sessions"]
+	maxAge = 0
+
 [plugins]
 	allowRemoteAdmin = true
 
-- 
2.21.0

